src/module.jl

```diff
@@ -18,7 +18,7 @@ type CuModule
     If the Julia debug level is 2 or higher (or, on 0.5, if CUDAdrv is loaded in DEBUG
     mode), line number and debug information will be requested when loading the PTX code.
     """
-    function CuModule(data, options::Dict{CUjit_option,Any}=Dict{CUjit_option,Any}())
+    function CuModule(data, options::Dict{CUjit_option,Any}=Dict{CUjit_option,Any}(); ctx = CuCurrentContext())
```

Comment:
Why do you do this? We do an API call to load the module data in this ctor, so it can't be that the context is anything else than the default one?